### ISSUE TITLE
Missing documentation: `Big.NE` and `Big.PE` not working

### ISSUE THREAD

**Issue by ackvf** [MikeMcl/big.js#93]
After an hour of labor I found out that they were renamed to `Big.E_NEG` and `Big.E_POS`.

Can you update the docs?

**Comment by MikeMcl** [comment]
I don't know what version of *big.js* you are using, but you have it the wrong way round - `Big.E_NEG` and `Big.E_POS` were renamed to `Big.NE` and Big.PE`, so the documentation *is* up to date.

**Comment by ackvf** [comment]
Now that's interesting. Actually I first installed some other library for big numbers and then I noticed I already had big.js installed probably as a sub-dependency, so to not bloat my dependencies I looked it up and then installed it as a local dependency. The version in package.json says `big.js": "^5.0.3",`. Also in `big.js/big.js` I can see a comment saying `v5.0.3`. Using command `big.version` prints `5.0.2`.
Not sure what was the problem yesterday, but when I checked today, it was OK.
Closing, thanks for info.
